### ISSUE TITLE
AutoDrive Development status update

### ISSUE THREAD

**Issue by Stephan-S** [Stephan-S/FS19_AutoDrive#273]
Hello to everyone active in this github repository.
Thank you for all your input, suggestions and feedback on this mod. It was tremendously helpful and important and AutoDrive would not be at this state without it!

I think most features of AutoDrive work as expected and it certainly has more features than intended at the start of development.
Of course there is still room for improvement and bugfixes as can be seen in the currently open issues, but overall I am quite happy that AutoDrive has reached the current stage.

Maybe some or most of you will have noticed that my time for Autodrive development, bugfixing and general responses to issues was very limited in the last weeks. And in this issue I would like to 'officially' state that I am currently not actively developing Autodrive any further. As stated above, all features are included in the mod that were supposed to be in it from the start. And if anyone wishes to add to those features, I am happy to merge pull requests and zip new release files.

So basically what I am trying to say is that at the current time I would like to pass on the development to anyone who wishes to contribute and I will probably come back fulltime for the next FS version and get the mod up and running for it when the time comes.

That does not mean that I am abandoning the project. I am still checking in, reading every mail to every issue and maybe once in a while write a small bugfix here and there, but I will not respond to every issue as I used to and I will probably not fix issues that are related to mod/vehicle/map compatibilities.

So do not expect new features from me, but build them yourself if you are up to the challenge :-)

Thanks to everyone again!

**Comment by Jamal467** [comment]
danke für die Leistung bis jetzt auch wen es noch 1 jahr dauert bleib bitte dran! greetz

**Comment by Bandesss** [comment]
Das ist sehr schade ls ohne dein mod geht garnicht. Danke für die großartige hifle

**Comment by Gariot** [comment]
Schon sehr schade, aber immerhin funktioniert die Mod weitestgehend :) Dafuer ein dickes fettes Danke! Du hast da echt tolles geleistet und mich fast komplett von Courseplay weggebracht. Das meiste macht jetzt einfach AD und der rest der normale helfer. Nur noch sehr spezielle faelle lasse ich von CP erledigen :)

Leider kenne ich mich mit LUA genau 0 aus, darum kann ich nur ein bisschen "support" bei den tickets betreiben :D Aber besser als nichts. Also dann!

Danke nochmal fuer deine Arbeit!

**Comment by sootysax** [comment]
Disappointing - the collaboration between courseplay and AD seemed promising, but without active development of AD, I feel like it will never really work well together.

**Comment by fudoido** [comment]
@sootysax i keep the same ideia i posted on CP github, CP should just take over AD and with the help of stephan, drop theyr course recording ay and use stephans, which is way more simple to use and works very good, including the braking next to curves and conection points which courseplay doesnt do.
and cp would only use their course creation for fieldwork and other minor stuff.
that way it would all be assembled on one single mod, but having the best of 2 mods toghther.

**Comment by koppi99** [comment]
danke für deine großartige Arbeit! bin schon gespannt auf den nächsten FS :)

**Comment by BrewingCoder** [comment]
I concur with the CP but CP's fieldwork for combines is a totally mess right now.  I realize how complicated it is but their combi unloader methods are just crap.    The only thing I trust CP to do these days is basically a single combine on a field that's so uneven a regular helper wouldn't be able to handle it.

**Comment by Bobster82** [comment]
> I concur with the CP but CP's fieldwork for combines is a totally mess right now. I realize how complicated it is but their combi unloader methods are just crap. The only thing I trust CP to do these days is basically a single combine on a field that's so uneven a regular helper wouldn't be able to handle it.

A bit off-topic, but ok...
@BrewingCoder , You know that mode2 on CP is still work in progress? As that mode is pretty complicated ;)
